nextcord/ui/view.py

```diff
@@ -168,18 +175,27 @@ class View:
     def __init_subclass__(cls) -> None:
         children: List[ItemCallbackType] = []
         for base in reversed(cls.__mro__):
-            for member in base.__dict__.values():
-                if hasattr(member, "__discord_ui_model_type__"):
-                    children.append(member)
+            children.extend(
+                member
+                for member in base.__dict__.values()
+                if hasattr(member, "__discord_ui_model_type__")
+            )
 
         if len(children) > 25:
             raise TypeError("View cannot have more than 25 children")
 
         cls.__view_children_items__ = children
 
-    def __init__(self, *, timeout: Optional[float] = 180.0, auto_defer: bool = True):
+    def __init__(
+        self,
+        *,
+        timeout: Optional[float] = 180.0,
+        auto_defer: bool = True,
+        prevent_update: bool = True,
+    ):
         self.timeout = timeout
         self.auto_defer = auto_defer
+        self.prevent_update = True if timeout else prevent_update
```

Comment:
Simply changing the default value for `prevent_update` and then **also** modifying the if statement so you in fact have not changed the default for this variable when you said you had is not cool.